fly-region/main.tf

```diff
@@ -0,0 +1,287 @@
+terraform {
+  required_version = ">= 1.0"
+
+  required_providers {
+    coder = {
+      source  = "coder/coder"
+      version = ">= 0.12"
+    }
+  }
+}
+
+variable "display_name" {
+    default = "Fly.io Region"
+    description = "The display name of the parameter."
+    type = string
+}
+
+variable "description" {
+    default = "The region to deploy workspace infrastructure."
+    description = "The description of the parameter."
+    type = string
+}
+
+variable "default" {
+    default = "ams"
```

Comment:
Can you make the default value of default null and use a condition in coder_paramter to set default as null if var.default is null

Reply:
Can do, the coder parameter default should already host the logic to do so. It checks if the default is in the filtered regions.